### PR TITLE
Alternative pivot table syntax

### PR DESCRIPTION
**Pull request by dustinmichels**
@carolynryan20:

This whole file is excellent, and this function call really does looks great. I believe the `pivot table` method allows you to do the same thing a little more "automatically." I don't actually care which version we use, I just thought you might like to see this alternative style!

### CODE REVIEW

File: marthas_dashboard/analysis/Association_Rules_Orange.py [hunk 1]
```diff
@@ -26,10 +26,7 @@ def getDFBuildingValuesInRange(building_id, start_date_time, stop_date_time):
     # data = api.building_values_in_range('4', '2016-08-18', '2017-08-19')
     df = api.building_values_in_range(building_id, start_date_time, stop_date_time)
 
-    df_pivot = (df
-              .groupby(['pointtimestamp', 'pointname'])['pointvalue']
-              .sum().unstack().reset_index().fillna(0)
-              .set_index('pointtimestamp'))
+    df_pivot = df.pivot_table(values='pointvalue', index='pointtimestamp', columns='pointname')
```
Comment (carolynryan20) [inline marthas_dashboard/analysis/Association_Rules_Orange.py]: Yes, I did see this! The reason that it got changed to this sort of pivoting was for something in the other algorithm that we were trying (the apriori example), but if this works for Orange, It's definitely cleaner!

Comment (dustinmichels) [reply marthas_dashboard/analysis/Association_Rules_Orange.py]: Gotcha! As far as I can tell, the two lines are equivalent in this context:

*OLD:*
screen shot 2018-01-23 at 1 42 40 pm

*NEW:*
screen shot 2018-01-23 at 1 40 11 pm

But maybe you needed slightly more control in the apriori context?

Comment (dustinmichels) [reply marthas_dashboard/analysis/Association_Rules_Orange.py]: And again, it doesn't really matter. I just think its fun to exchange Pandas tricks!

**Comment by dustinmichels** [comment]
I'll close for now. Maybe we'll change it later.

**Pull request closed without merging**